apps/src/sites/code.org/pages/public/learn/localControlList.jsx

```diff
@@ -0,0 +1,85 @@
+import PropTypes from 'prop-types';
```

Comment:
nit: can you move this in `apps/src/templates` and call in `LocalControlList`? Sorry to be picky, it just matches other files a bit better. I think this should be just moving and adjusting imports but let me know if it gets more complex for any reason!